packages/debugger/test/data/returnvalue.js

```diff
@@ -297,7 +297,7 @@ describe("Return value decoding", function () {
     let instance = await abstractions.ReturnValues.deployed();
     let txHash;
     try {
-      await instance.fail(); //web3 throws on failure
+      await instance.fail({ gas: gasLimit }); //web3 throws on failure
```

Comment:
Do we only deal with `gas`? Should we eventually handle `gasLimit`? [Here](https://ethereum.org/en/developers/docs/transactions/) in the params section they list `gasLimit` and not `gas`.

Reply:
Regarding the first, Truffle and web3 call it `gas`.  That's surprising that that documentation calls it `gasLimit`.  I've never seen it called that before.

As for the second, yes, it is.  Why this value?  I dunno, but honestly, it doesn't particularly matter.  I left the actual value unchanged from what it already was.

Reply:
Maybe we should set another constant named something like `lowTxGasLimit` or something to avoid future confusion eh?

Reply:
Yeah to my knowledge it doesn't matter... I think `gasLimit` is probably fine as a name?  Calling it `arbitraryGasLimit` doesn't sound to me like it really adds much meaningful.

Reply:
Yeah but I kind of feel like people (including myself) get confused enough with the gas thing as it is that it is worth making it a separate case. I guess if you really don't want to do it I'll back off.

Reply:
I guess I could rename it to `blockGasLimit`, if you think that would be clearer?  Of course it would then *also* be used for the transaction gas limit, but that does at least make it more explicit that we *are* reusing the block gas limit as the tx gas limit.  Does that sound good to you?

Otherwise I'd rather leave it alone; it's just test code, after all.

Reply:
Why couldn't we just export two variables with different names even if they have the same value so it is clear exactly what each value is for? Doesn't it kind of seem hacky to re-use a block gas limit constant for a tx gas limit? I mean, those two terms have similar names but they are very different things.

I know it is just test code but a lot of people are going to be working on this in the future and I can foresee confusion when trying to reason about tests.

Reply:
> Why couldn't we just export two variables with different names even if they have the same value so it is clear exactly what each value is for?

Hm, I guess I could go with that?  Seems a bit more than is necessary, but it is at least explicit.  I'll go do that I guess.

Reply:
thanks @haltman-at!

Reply:
OK, I've gone ahead and made this change.